src/test/java/com/dynatrace/openkit/protocol/BeaconTest.java

```diff
@@ -1132,7 +1132,7 @@ public void sessionIDIsAlwaysValue1OnDataCollectionLevel1() {
         target.setBeaconConfiguration(new BeaconConfiguration(1, DataCollectionLevel.PERFORMANCE, CrashReportingLevel.OFF));
 
         //when
-        int sessionID = target.getSessionNumber(SESSION_ID);
+        int sessionID = target.getSessionNumber();
```

Comment:
rename for consistency reasons (below `sessionID` was renamed to `sessionNumber`)